app/views/collections/_sidebar.html.erb

```diff
@@ -48,8 +48,13 @@
       <li><%= span_if_current(ts('Manage Items', key: 'dashboard'), collection_items_path(@collection)) %></li>
       <% if @collection.user_is_owner?(current_user) %>
         <% if @collection.challenge %>
-          <li><%= span_if_current ts('Challenge Settings', key: 'dashboard'), eval("edit_collection_#{challenge_class_name(@collection)}_path(@collection)") %></li>
-      	<% end %>
+          <% if @collection.prompt_meme? %>
+            <li><%= span_if_current ts('Challenge Settings', key: 'dashboard'), edit_collection_prompt_meme_path(@collection) %></li>
```

Comment:
Why this approach instead of

if @collection.prompt_meme?
  yada
elsif @collection.gift_exchange?
  yada
end

Reply:
When I parse elsif I always have to look up what it does, while a set of ifs where the conditions are exclusive is clear to me.

Reply:
Ah, okay. Please change to elsif, then!